### DISCUSSION TITLE
Custom Markup - Comments (Tony)

### DISCUSSION THREAD

**Discussion by AnthonyMuscio** [wikilabs/plugins#93] [Custom Markup]
Folks,

Is it ok if I run my own discussion and comments here?

As I return to Custom Markup I am struggling with some basic issues, and would like to raise issues and discuss some things without being wrapped up in all the other issues.

**Observations**

1. The Custom Markup definition](https://wikilabs.github.io/editions/custom-markup/#Custom%20Markup%20Definition) names but does not detail the _endString Parameter. Are \n and \n\n valid symbolic end strings we can use?
2. If we have text then a standard wikitext it will be ignored and wrapped
```
text
*text2
```
resulting in
`text *text`

Is there any way in custom markup to cause the "\n" of the previous line be enough to make glyph be enough?

3.

**Comment by pmario** [comment]
`_endString` not in the docs .... I'll fix that.

> Are \n and \n\n valid symbolic end strings we can use?

No. They are used by default. If you want to use them, don't add the parameter.

---------

Do you intend to "redefine" the defaults? ... I'll have to think about that.

**Reply by pmario** [reply]
see: https://github.com/wikilabs/plugins/issues/94

**Comment by pmario** [comment]
add 2

```
´ text
° text
* text2
```

The margin between the 2 texts are caused by the default CSS for `wltc`

**Reply by TiddlyTweeter** [reply]
> add 2
> 
> ```
> ´ text
> ° text
> * text2
> ```
> 
> The margin between the 2 texts are caused by the default CSS for `wltc`

Right. @AnthonyMuscio the key distinction, I think, is between **parser hard-coded behavior** that will effect what happens in the editor (blocks in entered text are established and managed by the parser) and **CSS in render** that can change spacing & looks.
